### ISSUE TITLE
Update version strings

### ISSUE THREAD

**Issue by etcinit** [etcinit/phabulous#51]
Figure out a better way of including the version in the binary. Currently, it is being hardcoded in a couple of places.

**Comment by perrohunter** [comment]
perhaps related to this, im not sure, is that the latest release `3.0.0-beta1` returns as if it was version 2.4.0 but the bot announces itself as if it was version 3
```
curl http://localhost:8085/
{"messages":["Welcome to the Phabulous API"],"status":"success","version":"2.4.0-beta1"}
```

**Comment by etcinit** [comment]
Yep. Looking to find a simple way to have it centralized in one place.

**Comment by holmboe** [comment]
I am also affected by this with the `v3.0.0-beta3` tag on quay.io.

**Comment by holmboe** [comment]
It is better to hardcode something like `UNKNOWN` in [front.go](/app/controllers/front.go), no? If tit is not guaranteed to be correct and updated then I suggest that it would be better.
